fixtures/stringify/src/lib.rs

```diff
@@ -0,0 +1,34 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+use paste::paste;
+
+uniffi::setup_scaffolding!();
+
+macro_rules! define_function {
+    ($type:ty) => {
+        paste! {
+            #[uniffi::export]
+            fn [<to_string_ $type>](value: $type) -> String {
+                value.to_string()
+            }
```

Comment:
I guess this is not used in the tests anymore, since you mentioned that stringifying is not an option

Reply:
ramoved